src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const nameChecking = age
+    ? <p className="Person__age">{`I am ${age}`}</p>
+    : null;
+
+  const marrigeCheck = sex === 'm'
```

Comment:
The variable name 'marrigeCheck' contains a typo and is not descriptive enough. Consider renaming it to 'marriageStatus' or something similar.